node/tests/RedisClient.test.ts

```diff
@@ -266,6 +266,31 @@ describe("RedisClient", () => {
         },
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "object refcount transaction test_%p",
+        async (protocol) => {
+            const client = await RedisClient.createClient(
+                getClientConfigurationOption(cluster.getAddresses(), protocol),
+            );
+
+            const key = uuidv4();
+            const transaction = new Transaction();
+            transaction.set(key, "foo");
+            transaction.objectRefcount(key);
+
+            const response = await client.exec(transaction);
+            expect(response).not.toBeNull();
+
+            if (response != null) {
```

Comment:
why is this `if` statement here?  Don't we check if response is not null in the previous line?

Reply:
Good point, I copy-pasted this from the other object commands but looks like it isn't necessary, I'll remove it in all the tests

Reply:
Hmm actually, it looks like js doesn't recognize that the assert makes it safe to access the response. If I remove this "if" I get this error:
`tests/RedisClient.test.ts:206:24 - error TS18047: 'response' is possibly 'null'.`